src/components/sets/reducer.js

```diff
@@ -571,9 +574,11 @@ function nodeTransformChildOrAppendChild(node,
  * @param {array} cellIds The new set value.
  * @param {string} name The name for the .isCurrent node. Optional.
  * By default, 'Current selection'.
+ * @param {boolean} hasProb Are the `cellIds` a list of string,
+ * or a list of [cellId, probability] tuples?
  * @returns {object} The updated tree.
  */
-function treeSetCurrentSet(currTree, cellIds, name = CURRENT_SELECTION_NAME) {
+function treeSetCurrentSet(currTree, cellIds, name = CURRENT_SELECTION_NAME, hasProb = false) {
```

Comment:
Is `hasProb` used anywhere as an argument where this function is used?

Reply:
No, this is outdated, thanks for catching this